docs/reference/api/docker_remote_api_v1.15.md

```diff
@@ -562,6 +564,7 @@ Json Parameters:
       a boolean value.
 -   **Dns** - A list of dns servers for the container to use.
 -   **DnsSearch** - A list of DNS search domains
+-   **DnsOptions** - A list of DNS options
```

Comment:
no, you're probably right that it shouldn't.  I am not very familiar with the internal factoring and I was not sure if it would show up as an additive field or not.